telemetry.js

```diff
@@ -0,0 +1,51 @@
+require('dotenv').config();
+
+const express = require('express');
```

Comment:
express is okayish, but by modern standards is a bit outdated.. `koa` and now `fastify` are considered better